### ISSUE TITLE
No access to stored custom radio stations, alternative not functional

### ISSUE THREAD

**Issue by Isegrimm** [amp64/sonosbugtracker#26]
In the new app there is no way to access previously stored custom radio stations.

Using the desktop app, new radio stations can still be added and could also be accessed via the old TuneIn service. However, even though the old TuneIn service can be added, it is removed hours of minutes later, or it isn't even listed after adding.
Effectively. stations you add via the desktop client end up in limbo with no way of accessing them, unless using 3rd party apps.

The replacement Sonos suggests, which is to re-add all your stations to your TuneIn account so that they show up in the TuneIn (new) service, only shows around 30 of the favorites saved in your TuneIn account. I have 70-80 favorites, but I only see a random selection of around 30 of them in the Sonos app.

The webapp is even worse in that aspect, because there only 12 favorites are shown, and even less if your browser window is not wide enough.

**Comment by amp64** [comment]
I found that my custom stations were actually deleted from the Sonos graph when I installed the latest Desktop app on Windows, which really pissed me off.

**Comment by Isegrimm** [comment]
All your stations are gone? Even when using a 3rd party app? I use SonoPhone (iOS, paid) which still displays all my custom stations. So in my case the stations are still there, they’re just made inaccessible using the Sonos app.

I must say that a portion of my stations was deleted as well around 2 months ago. I used to have 350-ish stations, and there are around 70 of them missing. This might have happened after one of the updates Sonos pushed for either the firmware or the app, I’m not sure.

**Comment by furacao14** [comment]
@Isegrimm, fyi @amp64 is the SonoPhone creator ;-)

**Comment by amp64** [comment]
No, I am not the SonoPhone creator, I am the Phonos app creator :-) 

Custom radio stations are stored at R:0/0 in the UPnP tree and they were all deleted when I installed the latest PC Desktop app (which was the first version to block TuneIn (Classic)).

**Comment by furacao14** [comment]
@amp64, sorry about the confusion.
